src/trajectory_data_handling/sqlite_querying.hpp

```diff
@@ -15,8 +15,10 @@
 namespace trajectory_data_handling {
     enum query_purpose {
```

Comment:
Lav den til en enum class, når den kun tilhører namespacet. Det kræver et par ændringer i jeres switch, men jeg har sat kommentarer til det.

```suggestion
    enum class query_purpose {
```